master/buildbot/secrets/providers/file.py

```diff
@@ -58,22 +58,25 @@ def loadSecrets(self, dirname, suffixes):
                         secretvalue = source.read()
                     if suffix:
                         secretfile = secretfile[:-len(suffix)]
+                    if strip:
+                        secretvalue = secretvalue.rstrip("\r\n")
```

Comment:
That would not work with passwords that start with a space (yes, I have at least one such password).